packages/transformer/src/test/standalone/IModelTransformer.test.ts

```diff
@@ -2579,6 +2579,7 @@ describe("IModelTransformer", () => {
     const exporter = new IModelExporter(sourceDb, DetachedExportElementAspectsStrategy);
     const transformer = new IModelTransformer(exporter, targetDb, {
       includeSourceProvenance: true,
+      forceExternalSourceAspectProvenance: true
```

Comment:
Without it this test fails with 


` 1) IModelTransformer
       should transform all aspects when detachedAspectProcessing is turned on:

      AssertionError: expected 5 to equal 6
      + expected - actual

      -5
      +6

      at D:\imodel-transformer\packages\transformer\src\test\standalone\IModelTransformer.test.ts:2597:42
      at Set.forEach (<anonymous>)
      at Context.<anonymous> (src\test\standalone\IModelTransformer.test.ts:2591:16)`

Reply:
hmmm. I am guessing the 6th is the new ESA from the new provenance form and you can just change that expect to 5 and remove the new usage. Please confirm this

Reply:
went through this test together, it's none of that. this test has probably been broken for a bit on this branch due to it not asserting the expected subject count. @nick4598 will investigate further

Reply:
I changed detectElementDeletes to use isValidId64. Since the test sets up aspects on the root subject which is also the targetscopeelementId and it sets the identifier as 0 1 2 3 4, those aspects were being used to see if the element was deleted. isValidId64 allows those values to be skipped since it requires '0x' to be present.

The test also does not need + 1 as long as we don't forceEsaProv.

Could you explain what you mean by 'new provenance method'? I had assumed the new prov method was fed guids, and the old prov method is esas (I know there are still some ESAs in the new prov method). Would it be more accurate to say the 6th ESA is from setting forceExternalSouceAspectProvenance to true? These subjects should have fedguids and not require a 6th ESA as a result, right? 

I know forceEsaProv was not true before I opened this PR, but +1 also was never necessary and never being tested due to the way detectElementDeletes was working. But I'm guessing you were assuming the test was fully working previously and thats what made you go down the line of thought of 'new ESA' but I don't quite get the 'from the new provenance method' part.

Reply:
new ESA from scoping is what was meant